packages/roosterjs-editor-dom/lib/edit/pasteTable.ts

```diff
@@ -0,0 +1,71 @@
+import findClosestElementAncestor from 'roosterjs-editor-dom/lib/utils/findClosestElementAncestor';
+import getTagOfNode from 'roosterjs-editor-dom/lib/utils/getTagOfNode';
+import toArray from 'roosterjs-editor-dom/lib/jsUtils/toArray';
+import VTable from 'roosterjs-editor-dom/lib/table/VTable';
+import { NodePosition, NodeType, TableOperation } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ * STILL NOT WRITTEN
+ *
+ */
+export default function pasteTable(
+    root: HTMLElement,
+    nodeToInsert: Node,
+    position: NodePosition,
+    range: Range
+) {
+    if (
+        (nodeToInsert.childNodes.length >= 1 &&
```

Comment:
If length > 1, it means there will be other content than table to be pasted. 

I tried in Word in that case, it will directly insert whatever we copied, to have a nested table. I think we can follow this behavior.